anoncreds/src/services/verifier.rs

```diff
@@ -1197,7 +1214,25 @@ mod tests {
             },
         );
         res.insert(
-            "referent_2".to_string(),
+            "referent_before_from".to_string(),
+            Identifier {
+                timestamp: Some(1),
+                schema_id: SchemaId::default(),
```

Comment:
Huh, did not know I implemented default for the anoncreds ids. Ah, could be nice for testing :).